### DISCUSSION TITLE
pass the certification

### DISCUSSION THREAD

**Discussion by LUWY-116** [community/community#141552] [GitHub Education]
### Hi everyone,

I don't know what to ask in my application interface, I can only choose to take a photo, but not upload, which leads to a blurry material and I have not been able to pass the certification. But I'm really a college student!
![1728997142011](https://github.com/user-attachments/assets/5e3be8b0-b8ea-409e-80ea-8963f745917d)

**Comment by queenofcorgis** [comment]
@LUWY-116 have you tried accessing this window on another device or browser to see if you do have the upload button?

**Reply by LUWY-116** [reply]
sure,but it is useless.
